soroban-env-host/src/budget.rs

```diff
@@ -575,12 +645,21 @@ impl Default for BudgetImpl {
             // define the cpu cost model parameters
             let cpu = &mut b.cpu_insns.get_cost_model_mut(ct);
             match ct {
+                // This is the host cpu insn cost per wasm "fuel". Every "base" wasm
+                // instruction costs 1 fuel (by default), and some particular types of
+                // instructions may cost additional amount of fuel based on
+                // wasmi's config setting.
                 ContractCostType::WasmInsnExec => {
-                    cpu.const_term = 22;
-                    cpu.linear_term = 0;
-                }
+                    cpu.const_term = 0;
+                    cpu.linear_term = 22; // this is calibrated
```

Comment:
I just wanted to clarify how exactly this works. Will this number be multiplied by the fuel cost measured by wasmi at `wasmi_fuel_costs` rates?

Reply:
Yes, this will be multiplied by the `Fuel` numbers wasmi reports. So essentially this number is now "cpu_insns per fuel" and will be what we calibrate for.